src/NzbDrone.Core/Notifications/Webhook/Webhook.cs

```diff
@@ -0,0 +1,62 @@
+﻿
+using System.Collections.Generic;
+using FluentValidation.Results;
+using NzbDrone.Core.Tv;
+using NzbDrone.Common.Extensions;
+
+namespace NzbDrone.Core.Notifications.Webhook
+{
+    public class Webhook : NotificationBase<WebhookSettings>
+    {
+        private readonly IWebhookService _service;
+
+        public Webhook(IWebhookService service)
+        {
+            _service = service;
+        }
+
+        public override string Link
+        {
+            get { return "https://github.com/Sonarr/Sonarr/wiki/Webhook"; }
+        }
+
+        public override void OnGrab(string message)
+        {
+        }
+
+        public override void OnDownload(DownloadMessage message)
+        {
+            _service.OnDownload(message.Series, message.EpisodeFile, Settings);
+        }
+
+        public override void OnRename(Series series)
+        {
+            _service.OnRename(series, Settings);
+        }
+
+        public override bool SupportsOnGrab
+        {
+            get
+            {
+                return false;
```

Comment:
I think it would make sense to leave it in, for example if connecting it to slack or an IRC bot you might want to be notified of new grabs.